README.md

```diff
@@ -1,7 +1,11 @@
 # Quick install
 
-Run below command to install latest version of aeternity node.
+Run below command to install latest version of aeternity node:
+```bash
+curl -O https://raw.githubusercontent.com/aeternity/installer/master/install.sh && chmod +x install.sh && ./install.sh
```

Comment:
We need to clarify what the plan is re https://github.com/aeternity/aeternity/blob/1cf57c6a51dc0b24d65d9d651522f328f259171e/docs/installation.md#quick-install

Reply:
I'd probably propose changes to drop the node version anytime soon